stubmodify/modify_test.go

```diff
@@ -68,7 +68,7 @@ func TestWriteAttributionCodeFull(t *testing.T) {
 	copy(origBytes, fileBytes)
```

Comment:
What was changed in `testdata/test-stub.exe`?

Reply:
I inserted non-zero data after the `__MOZCUSTOM__:` tag (I believe that all of the 1024 bytes that could be used in attribution are now non-zero).